civictechindexadmin/data/models.py

```diff
@@ -81,6 +81,21 @@ def __str__(self):
         return f"Alias {self.id}: {self.tag} {self.alias}"
 
 
+class Contact(models.Model):
```

Comment:
You added a model but where is the migration? You can create it using `./manage.py makemigrations`. The name generated automatically may or may not say 'added_contact_model'. If you end up with on with a name like "0004_auto_<datestamp>" please rename it to be more descriptive.

I am puzzled by the failing tests in this PR. Hopefully adding a migration file will fix - or at least give us a more understandable error.